core/build.gradle.kts

```diff
@@ -59,6 +67,19 @@ val buildCInteropDef by tasks.registering {
     outputs.files(defFile)
 }
 
+tasks.create<Exec>("buildNative") {
```

Comment:
Run `build.sh` to build the binary and save it to `build/native/powersync-sqlite/${OS}/output`